astropy/utils/decorators.py

```diff
@@ -36,6 +49,7 @@ def deprecated(
     pending=False,
     obj_type=None,
     warning_type=AstropyDeprecationWarning,
+    pending_warning_type=AstropyPendingDeprecationWarning,
```

Comment:
It's too late for existing arguments, but I'd suggest making any new argument in public functions keyword-only (not crucial, but helps with deprecation cycles, which this PR is all about 😄)
```suggestion
    *,
    pending_warning_type=AstropyPendingDeprecationWarning,
```

Reply:
Fixed.